src/com/google-map.js

```diff
@@ -2,17 +2,26 @@
  * @author Deux Huit Huit
  *
  * Google maps component
+ *
+ * It needs to have the API key in the DOM as
+ * <script data-gmap-key="<YOUR KEY>"></script>
  */
 (function ($, win, global, undefined) {
 
 	'use strict';
 	
 	App.components.exports('googleMap', function (o) {
-		
+		var MAX_RETRIES = 40;
+		var scope;
 		var container;
 		var markers = [];
 		var map;
+		var mapIsReady = false;
 		var openedMarker;
+		var urlGoogleMaps = 'https://maps.googleapis.com/maps/api/js?key=' +
+			$('[data-gmap-key]').attr('data-gmap-key');
```

Comment:
May break if multiple data-gmap-key attribute are present

Reply:
Add a first() just in case?

Reply:
@dominic-blain .attr('') already does it for us. .val() also